BlazorUI/Program.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace BlazorUI
+{
+    public class Program
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)